src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -100,6 +127,38 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService,
         return ResponseEntity.ok(participation);
     }
 
+    @GetMapping("/exercise/{exerciseId}/submission/{submissionId}/exampleAssessment")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<Result> getExampleAssessmentForTutor(@PathVariable Long exerciseId, @PathVariable Long submissionId) {
+        log.debug("REST request to get example assessment for tutors text assessment: {}", submissionId);
+        Optional<TextSubmission> textSubmission = textSubmissionRepository.findById(submissionId);
+        TextExercise textExercise = textExerciseService.findOne(exerciseId);
+        if (!textSubmission.isPresent()) {
+            return ResponseEntity.badRequest().headers(HeaderUtil.createFailureAlert("textSubmission", "textSubmissionNotFound", "No Submission was found for the given ID.")).body(null);
+        }
+
+        // If the user is not an instructor, and this is not an example submission used for tutorial,
+        // do not provide the results
+        boolean isAtLeastTeachingAssistant = authCheckService.isAtLeastTeachingAssistantForExercise(textExercise);
+        if (!textSubmission.get().isExampleSubmission() && !isAtLeastTeachingAssistant) {
```

Comment:
Do you really want to have `&&` here? Sounds rather like `||`

Reply:
I think `&&` is fine: to deny the request, it has to not be an example submission, and the user has to not be an instructor.

If it is an example submission we want always to return it, also if the user is not an instructor

## Cases

| Role       | ExampleSubmission? | Want to deny? | Deny with &&                          | Deny with  or                         |
|------------|--------------------|---------------|---------------------------------------|---------------------------------------|
| Tutor      | Yes                | No            | No (!isExampleSubmission() is false)  | Yes (!isAtLeastInstructor is true)    |
| Tutor      | No                 | Yes           | Yes (both conditions are true)        | Yes                                   |
| Instructor | Yes                | No            | No (!isAtLeastInstructor() is false)  | No                                    |
| Instructor | No                 | No            | No ( !isAtLeastInstructor() is false) | Yes (!isExampleSubmission() is false) |